database/migrations/2022_05_13_115716_email.php

```diff
@@ -13,10 +13,8 @@
      */
     public function up()
     {
-        Schema::create('password_resets', function (Blueprint $table) {
```

Comment:
Kāpēc Jūs pārveidojāt migrāciju no stadarta versijas uz jaunu?

Reply:
Ja godīgi, neatceros.
Tik daudzas reies nekas nemigrēja un databāzē neparādījās, ka es dažreiz atjaunoju visas vecās migrācijas, kuras gribēju izdzēst.
Rādījās dažādi laravel paziņojumi un errori, un es nebiju pārliecināta, ka drīkstēju izdzēst standarta versijas.
Tad mēģināju tās izveidot par jaunu 0 droši vien.
Neatceros.
Vardu sakot - kaut kāds haoss pagaidām no manas puses.)